Description: call rsvg-convert instead of rsvg
Author: Scott Howard <showard@debian.org>
Bug: bugs.debian.org/659259
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/imagemagick/+bug/929573
--- imagemagick-6.6.9.7.orig/configure.ac
+++ imagemagick-6.6.9.7/configure.ac
@@ -2861,7 +2861,7 @@ else
 fi
 RLEEncodeDelegateDefault='rawtorle'
 RMDelegateDefault='rm'
-RSVGDecodeDelegateDefault='rsvg'
+RSVGDecodeDelegateDefault='rsvg-convert'
 SCANDecodeDelegateDefault='scanimage'
 TXTDelegateDefault='enscript'
 UniconvertorDelegateDefault='uniconvertor'
--- imagemagick-6.6.9.7.orig/configure
+++ imagemagick-6.6.9.7/configure
@@ -30578,7 +30578,7 @@ else
 fi
 RLEEncodeDelegateDefault='rawtorle'
 RMDelegateDefault='rm'
-RSVGDecodeDelegateDefault='rsvg'
+RSVGDecodeDelegateDefault='rsvg-convert'
 SCANDecodeDelegateDefault='scanimage'
 TXTDelegateDefault='enscript'
 UniconvertorDelegateDefault='uniconvertor'
--- imagemagick-6.6.9.7.orig/config/delegates.xml
+++ imagemagick-6.6.9.7/config/delegates.xml
@@ -102,7 +102,7 @@
   <delegate decode="scanx" command="&quot;scanimage&quot; &gt; &quot;%o&quot;"/>
   <delegate decode="png" encode="show" spawn="True" command="&quot;/usr/local/bin/display&quot; -delay 0 -window-group %[group] -title &quot;%l &quot; &quot;ephemeral:%i&quot;"/>
   <delegate decode="shtml" command="&quot;html2ps&quot; -U -o &quot;%o&quot; &quot;%i&quot;"/>
-  <delegate decode="svg" command="&quot;rsvg&quot; &quot;%i&quot; &quot;%o&quot;"/>
+  <delegate decode="svg" command="&quot;rsvg-convert&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
   <delegate decode="txt" encode="ps" mode="bi" command="&quot;enscript&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
   <delegate decode="png" encode="win" stealth="True" spawn="True" command="&quot;/usr/local/bin/display&quot; -immutable -delay 0 -window-group %[group] -title &quot;%l &quot; &quot;ephemeral:%i&quot;"/>
   <delegate decode="wmf" command="&quot;wmf2eps&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
--- imagemagick-6.6.9.7.orig/config/delegates.xml.in
+++ imagemagick-6.6.9.7/config/delegates.xml.in
@@ -102,7 +102,7 @@
   <delegate decode="scanx" command="&quot;@SCANDecodeDelegate@&quot; &gt; &quot;%o&quot;"/>
   <delegate decode="miff" encode="show" spawn="True" command="&quot;@DisplayDelegate@&quot; -delay 0 -window-group %[group] -title &quot;%l &quot; &quot;ephemeral:%i&quot;"/>
   <delegate decode="shtml" command="&quot;@HTMLDecodeDelegate@&quot; -U -o &quot;%o&quot; &quot;%i&quot;"/>
-  <delegate decode="svg" command="&quot;@RSVGDecodeDelegate@&quot; &quot;%i&quot; &quot;%o&quot;"/>
+  <delegate decode="svg" command="&quot;@RSVGDecodeDelegate@&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
   <delegate decode="txt" encode="ps" mode="bi" command="&quot;@TXTDelegate@&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
   <delegate decode="miff" encode="win" stealth="True" spawn="True" command="&quot;@DisplayDelegate@&quot; -immutable -delay 0 -window-group %[group] -title &quot;%l &quot; &quot;ephemeral:%i&quot;"/>
   <delegate decode="wmf" command="&quot;@WMFDecodeDelegate@&quot; -o &quot;%o&quot; &quot;%i&quot;"/>
