--- a/libpng-1.2.46.txt	2009-12-03 12:46:00.000000000 +1100
+++ b/libpng-1.2.46.txt	2009-12-04 11:59:32.000000000 +1100
@@ -1049,8 +1049,9 @@
     gray = (rw*red + gw*green + bw*blue)/65536;
 
 The default values approximate those recommended in the Charles
-Poynton's Color FAQ, <http://www.inforamp.net/~poynton/>
-Copyright (c) 1998-01-04 Charles Poynton <poynton at inforamp.net>
+Poynton's Color FAQ,
+<http://www.poynton.com/notes/colour_and_gamma/ColorFAQ.html>
+Copyright (c) 2006-11-28 Charles Poynton <poynton at poynton.com>
 
     Y = 0.212671 * R + 0.715160 * G + 0.072169 * B
 
--- a/png.h	2009-12-03 12:46:00.000000000 +1100
+++ b/png.h	2009-12-04 11:56:48.000000000 +1100
@@ -1859,15 +1859,17 @@
    png_ptr_ptr, png_infopp info_ptr_ptr, png_infopp end_info_ptr_ptr));
 
 /* Free all memory used by the read (old method - NOT DLL EXPORTED) */
-extern void png_read_destroy PNGARG((png_structp png_ptr, png_infop info_ptr,
-   png_infop end_info_ptr)) PNG_DEPRECATED;
+/* Debian note: exporting as it is required by legacy applications */
+extern PNG_EXPORT(void,png_read_destroy) PNGARG((png_structp png_ptr, png_infop info_ptr,
+   png_infop end_info_ptr));
 
 /* Free any memory associated with the png_struct and the png_info_structs */
 extern PNG_EXPORT(void,png_destroy_write_struct)
    PNGARG((png_structpp png_ptr_ptr, png_infopp info_ptr_ptr));
 
 /* Free any memory used in png_ptr struct (old method - NOT DLL EXPORTED) */
-extern void png_write_destroy PNGARG((png_structp png_ptr)) PNG_DEPRECATED;
+/* Debian note: exporting as it is required by legacy applications */
+extern PNG_EXPORT(void,png_write_destroy) PNGARG((png_structp png_ptr));
 
 /* Set the libpng method of handling chunk CRC errors */
 extern PNG_EXPORT(void,png_set_crc_action) PNGARG((png_structp png_ptr,
--- a/png.5	2009-12-03 12:46:00.000000000 +1100
+++ b/png.5	2009-12-04 11:59:32.000000000 +1100
@@ -18,7 +18,11 @@
 platforms.
 
 .SH "SEE ALSO"
-.IR libpng(3) ", " zlib(3) ", " deflate(5) ", and " zlib(5)
+.IR libpng (3),
+.IR zlib (3),
+.IR deflate (5),
+and
+.IR zlib (5)
 .LP
 PNG specification (second edition), November 2003:
 .IP
