--- a/src/fcname.c.orig	2008-09-12 17:27:25.000000000 +0000
+++ b/src/fcname.c	2008-09-12 17:31:54.000000000 +0000
@@ -441,5 +441,6 @@
     { (FcChar8 *) "lcddefault",	    "lcdfilter",    FC_LCD_DEFAULT },
     { (FcChar8 *) "lcdlight",	    "lcdfilter",    FC_LCD_LIGHT },
     { (FcChar8 *) "lcdlegacy",	    "lcdfilter",    FC_LCD_LEGACY },
+    { (FcChar8 *) "lcdfilterlegacy",	    "lcdfilter",    FC_LCD_LEGACY },
 };
 
