diff -Nur -x '*.orig' -x '*~' fontconfig-2.8.0/conf.d/11-lcd-filter-lcddefault.conf fontconfig-2.8.0.new/conf.d/11-lcd-filter-lcddefault.conf
--- fontconfig-2.8.0/conf.d/11-lcd-filter-lcddefault.conf	1969-12-31 19:00:00.000000000 -0500
+++ fontconfig-2.8.0.new/conf.d/11-lcd-filter-lcddefault.conf	2010-01-13 19:16:57.691482777 -0500
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
+<!-- conf.d/lcd-filter-lcddefault.conf -->
+<fontconfig>
+<!--  Use lcddefault as default for LCD filter -->
+  <match target="font">
+    <edit mode="assign" name="lcdfilter">
+      <const>lcddefault</const>
+    </edit>
+  </match>
+</fontconfig>
diff -Nur -x '*.orig' -x '*~' fontconfig-2.8.0/conf.d/Makefile.am fontconfig-2.8.0.new/conf.d/Makefile.am
--- fontconfig-2.8.0/conf.d/Makefile.am	2010-01-13 19:16:56.662484262 -0500
+++ fontconfig-2.8.0.new/conf.d/Makefile.am	2010-01-13 19:16:57.691482777 -0500
@@ -37,6 +37,7 @@
 	10-sub-pixel-vbgr.conf \
 	10-sub-pixel-vrgb.conf \
 	10-unhinted.conf \
+	11-lcd-filter-lcddefault.conf \
 	20-fix-globaladvance.conf \
 	20-unhint-small-vera.conf \
 	25-unhint-nonlatin.conf \
@@ -63,6 +64,7 @@
 	10-antialias.conf \
 	10-hinting.conf \
 	10-hinting-slight.conf \
+	11-lcd-filter-lcddefault.conf \
 	20-fix-globaladvance.conf \
 	20-unhint-small-vera.conf \
 	30-urw-aliases.conf \
diff -Nur -x '*.orig' -x '*~' fontconfig-2.8.0/conf.d/Makefile.in fontconfig-2.8.0.new/conf.d/Makefile.in
--- fontconfig-2.8.0/conf.d/Makefile.in	2010-01-13 19:16:56.662484262 -0500
+++ fontconfig-2.8.0.new/conf.d/Makefile.in	2010-01-13 19:16:57.691482777 -0500
@@ -264,6 +264,7 @@
 	10-sub-pixel-vbgr.conf \
 	10-sub-pixel-vrgb.conf \
 	10-unhinted.conf \
+	11-lcd-filter-lcddefault.conf \
 	20-fix-globaladvance.conf \
 	20-unhint-small-vera.conf \
 	25-unhint-nonlatin.conf \
@@ -289,6 +290,7 @@
  	10-antialias.conf \
  	10-hinting.conf \
  	10-hinting-slight.conf \
+	11-lcd-filter-lcddefault.conf \
 	20-fix-globaladvance.conf \
 	20-unhint-small-vera.conf \
 	30-urw-aliases.conf \
